react-client/src/Components/DomInfoDialogBox/index.js

```diff
@@ -62,8 +64,8 @@ const DomInfoDialogBox = ({ id, idx, tag, classNames, parent, children, top, lef
             });
 
             window.store.DomInfoDialogBox.children = newChildren; //need to put it inside window.store so it gets the updated children (see mutation type attributes)
-
-            setDomInfo({ ...domInfo, classNames: window.store.DomInfoDialogBox.classList, children: newChildren });
+            setDomInfo({ ...domInfo, classNames: window.store.DomInfoDialogBox.classList, children: domUtils.customChildFilter(newChildren) });
```

Comment:
This looks good. Thanks

Reply:
Had to revert back to original since this approach introduced error in the system i applied domutils on line 197 and made slight changes in dom utils